### DISCUSSION TITLE
Releases API - can't figure out order

### DISCUSSION THREAD

**Discussion by ctrlaltca** [community/community#21901] [New to GitHub]
<p>Hi, i can’t figure out how releases are sorted in the “releases” API.</p>
<p>As far as i understand it used to sort releases by “published_at” descendent, but now i found that in <a href="https://api.github.com/repos/Cockatrice/Cockatrice/releases" rel="nofollow noopener">https://api.github.com/repos/Cockatrice/Cockatrice/releases</a> the first listed release is older than the second release:</p>
<ol>
<li>
<p>version Development-2.5.0-beta.16 is dated: </p>
<p>“published_at”: “2018-02-19T08:01:27Z”</p>
</li>
</ol>
<p> 2. Release-2.5.0 is dated: </p>
<pre><code>"published_at": "2018-03-02T21:51:53Z"
</code></pre>
<p>If you look at the “releases” page instead: <a href="https://github.com/Cockatrice/Cockatrice/releases" rel="nofollow noopener">https://github.com/Cockatrice/Cockatrice/releases</a> , they are correctly sorted by date.</p>
<p>Did i wrongly assume that releases in the API page are sorted or did i stumble upon a strange behavior of he releases api?</p>
<p>Thank you</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There isn’t a guarantee of a sort order in the API documentation, so I wouldn’t assume that any particular sort order is guaranteed. It <em>looks</em> like it is still sorted by <code>published_at</code> with the exception that the <code>prerelease: true</code> record is first. So it is possible that it is ordered first by <code>prerelease</code> and then by <code>published_at</code>, but since there is no documented guaranteed sort order it is probably not wise to depend on that behavior.</p>
